src/SmartComponents/modules/__tests__/reducer-test.js

```diff
@@ -0,0 +1,97 @@
+import reducers from '../reducers';
+import types from '../types';
+
+describe('compare reducer', () => {
+    it('should return initial state', () => {
+        expect(reducers.compareReducer(undefined, {})).toEqual(
+            {
+                compare: {},
+                addSystemModalOpened: false,
+                selectedSystemIds: []
+            }
+        );
+    });
+
+    it('should handle FETCH_COMPARE_FULFILLED', () => {
+        expect(
+            reducers.compareReducer({}, {
+                facts: [
+                    {
+                        name: 'bios_uuid',
+                        state: 'SAME',
+                        systems:
+                            {
+                                '9c79efcc-8f9a-47c7-b0f2-142ff52e89e9': null,
+                                'f35b1e1d-d231-43f2-8e4f-8f9cb01e3aa2': null
+                            }
+                    },
+                    {
+                        name: 'display_name',
+                        state: 'SAME',
+                        systems:
+                            {
+                                '9c79efcc-8f9a-47c7-b0f2-142ff52e89e9': null,
+                                'f35b1e1d-d231-43f2-8e4f-8f9cb01e3aa2': null
+                            }
+                    }
+                ],
+                /* eslint-disable camelcase */
+                metadata: [
+                    {
+                        fdqn: 'sgi-xe500-01.rhts.eng.bos.redhat.com',
+                        id: '9c79efcc-8f9a-47c7-b0f2-142ff52e89e9',
+                        last_updated: '2019-01-15T14:53:15.886891Z'
+                    },
+                    {
+                        fdqn: 'ibm-x3650m4-03-vm03.lab.eng.brq.redhat.com',
+                        id: 'f35b1e1d-d231-43f2-8e4f-8f9cb01e3aa2',
+                        last_updated: '2019-01-15T15:25:16.304899Z'
+                    }
+                ],
+                /* eslint-enable camelcase */
+                type: types.FETCH_COMPARE
```

Comment:
I think this should be `${types.FETCH_COMPARE}_FULFILLED` given thats the action in the reducer.